.github/workflows/build.yml

```diff
@@ -0,0 +1,96 @@
+name: CI
+
+on:
+  push:
```

Comment:
Is this needed? The documentation of `pull_request` event [states](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request):
>workflow runs when a pull request is opened or reopened or when the head branch of the pull request is updated.

Reply:
Nice catch, will remove this. Otherwise, this CI will be trigger twice when a PR is merged into `master`:
- merge_group
- this

Reply:
updated